### DISCUSSION TITLE
Please add video tutorials into the docs

### DISCUSSION THREAD

**Discussion by ShaileshInfyom** [spatie/laravel-medialibrary#2217] [Show and tell] [answered]
This is a great package and very useful for manage media. I create a  video tutorial series of this package of how to install and use this package. so, I hope this tutorial is helpful to the installation and usages of the beginner. Thanks for this awesome gift for laravel community. So, please add look like https://spatie.be/docs/laravel-permission/v3/advanced-usage/ui-options 
here is the link https://www.youtube.com/playlist?list=PL0wCC44AhrC2yMytNoYeJtBikvkTCr7Nt please add it somewhere here
https://spatie.be/docs/laravel-medialibrary/v7/introduction

**Comment by freekmurze** [comment, marked as answer]
Hi, we already have video tutorials in our docs. I think having multiple ones could be confusion for users.
